accounts-db/src/buffered_reader.rs

```diff
@@ -179,6 +186,28 @@ impl<'a, const N: usize> BufferedReader<'a, Stack<N>> {
     }
 }
 
+/// Open file at `path` with buffering reader using `buf_size` memory and doing
+/// read-ahead IO reads (if `io_uring` is supported by the host)
+pub fn large_file_buf_reader(
+    path: impl AsRef<Path>,
+    buf_size: usize,
+) -> IoResult<Box<dyn BufRead>> {
```

Comment:
nit: imo IoResult is ugly, in other parts of the codebase we do io::Result which